proxies-es6/demo.js

```diff
@@ -0,0 +1,43 @@
+'use strict';
+
+// Proxying a normal object
+var target = {};
+var handler = {
+  get(receiver, name = 'world') {
```

Comment:
In addition to @beaufortfrancois's comment, can you double-check the `get` method signature? As per [MSDN](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Proxy/handler/get#Syntax), it's

```js
var p = new Proxy(target, {
  get: function(target, property, receiver) {
  }
});
```

You're using the [method definition shorthand](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Method_definitions), but I don't think that should have any effect on the parameter orders, unless there's something implicit going on?

I also think using `property` instead of `name` for that parameter makes things clearer.